composer.json

```diff
@@ -39,6 +42,7 @@
     "psr/http-message": "To use standard GraphQL server"
   },
   "scripts": {
-    "lint" : "phpcs"
+    "lint" : "phpcs",
+    "static-analysis": "@php bin/phpstan.phar analyse --ansi -l 1 -c phpstan.neon src"
```

Comment:
why use phar instead of standard binary?

`composer require --dev phpstan/phpstan` and then `vendor/bin/phpstan`

Reply:
stil valid